homeassistant/components/jewish_calendar/__init__.py

```diff
@@ -25,7 +25,7 @@
 CONF_CANDLE_LIGHT_MINUTES = "candle_lighting_minutes_before_sunset"
 CONF_HAVDALAH_OFFSET_MINUTES = "havdalah_minutes_after_sunset"
 DEFAULT_NAME = "Jewish Calendar"
-DEFAULT_CANDLE_LIGHT = 18
```

Comment:
Please do not change this in this PR. It's unrelated to moving to config flow.
Also as I mentioned in our offline discussion, 18 is the default in a lot of places in the diaspora.